### PR TITLE
Add recipe for dask_labextension

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review/merge.

File: recipes/dask_labextension/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: "c3fa82fd82a65970e5c65412762709cdbe4852c13ec71bbaff923ebc9cadaff8"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - python >= 3.6
```
Comment (chrisburr) [inline recipes/dask_labextension/meta.yaml]: There shouldn’t be a space between >= and 3.6.

File: recipes/dask_labextension/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+    # distributed and notebook are directly imported by setup.py
+    - distributed >=1.24.1
+    - notebook >=4.3.1
+  run:
+    - python >= 3.6
```
Comment (chrisburr) [inline recipes/dask_labextension/meta.yaml]: Also here

File: recipes/dask_labextension/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c3fa82fd82a65970e5c65412762709cdbe4852c13ec71bbaff923ebc9cadaff8"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (chrisburr) [inline recipes/dask_labextension/meta.yaml]: This can be simplified now to remove the no deps and ignore installed flags

**Comment by xhochy** [comment]
@chrisburr Can you have another look=

**Pull request merged**